docs/docs/software-architecture.md

```diff
@@ -31,139 +31,147 @@ UI - User Interface
+- Docker
 
-Blockchain Gateway:
+Blockchain Gateway:   
 
 - Bitcoin libraries:
-  - Bitcoinjs (junderw, 2020)
+  - Bitcoinjs-lib (junderw, 2020)
 - Bitshare libraries:
-  - Bitsharejs
+  - bitsharesjs
 
-HTLC/XXX:
+HTLC:   
 
 - Creates the HTLC on its own blockchain
 
-Database:
-
-- Mongoose/MongoDB
-
-Logging:
+Logging:   
 
 - To be decided by ChainSquad if it gets outsourced to a dedicated SaaS
 
+   
 ## 7. Implementation
 
 The implementation diagram is used to visualize the flow of control and the implementation description elaborates further on the processes taking place.
-
+   
 ### 7.1. Implementation Diagram
 
-![](./docs/static/img/newImplementation.svg)
+![](img/Implementation.svg)   
 Figure 5: Implementation Diagram (Swapchain, 2020) [4]
-
+   
 ### 7.2. Implementation Description
 
-End-User:
-The user interacts with the Swapchain UI without signing-up or authenticating an account. For now, we assume that proposer and taker already know each other. Next, both exchange partners have to fill out a form to either accept or propose an atomic cross-chain swap. The user initiates the atomic swap by submitting the respective order.
+End-User:   
+The user interacts with the Swapchain UI without signing-up or authenticating an account. For now, we assume that proposer and accepter already know each other. Next, both exchange partners have to fill out a form to either accept or propose an atomic cross-chain swap. The user initiates the atomic swap by submitting the respective form and therewith opening an HTLC.
 
-System:
-After the system receives the order submitted by the user a swap request is carried out using a HTLC. For that an API verifies the signatures and ensures that the private key matches the public key for the swap to take place. As soon as all required conditions are met it relays back and responds to the system to successfully transfer the funds. The involved users will receive a notification if the transaction was successful or not. In case the transaction fails, the users will receive a notification that they will be receiving the refund.
+System:   
+After the system receives the proposal created by the user, a swap request is carried out using an HTLC. Therefore, the signatures are verified so that the private key matches the public key for the swap to take place. As soon as all required conditions are met it relays back and responds to the system to successfully transfer the funds. The involved users will receive a error message if the transaction was unsuccessful. In this case the users will be receiving the refund.
 
+   
 ## 8. Size and Performance
 
 The chosen software architecture supports the following requirements:
 
-1. The system shall support two simultaneous users against the central database at any given time.
-2. The system shall be able to complete an ACCS transaction within 5 minutes once all HTLC requirements are met.
+1. The system shall support at least two simultaneous users at a time.
```

Comment:
We could easyily support thousands of requests at the same time becuase we are using netlifies CDN.
The only hard cap is 100GB of bandwidth a month on the free tier.